src/components/templates/FieldGroupTemplate/index.tsx

```diff
@@ -54,7 +53,7 @@ export default function FieldGroupTemplate(props) {
   pConn.setInheritedProp('displayMode', 'LABELS_LEFT');
   const memoisedReadOnlyList = useMemo(() => {
     return referenceList.map((item, index) => (
-      <FieldGroup item={item} key={index} name={`${HEADING} ${index + 1}`} />
+      <FieldGroup item={item} key={item.ProductName} name={`${HEADING} ${index + 1}`} />
```

Comment:
We shouldn't hardcode productName here as that might get changed for other data objects.

Reply:
Ok. So there's nothing we can readily do for now other than use index and turn off the warning?

Reply:
yes, as we are using the index as a key in the read-only element, it mostly won't create any problem for now